internal/xsql/sqlValidator.go

```diff
@@ -31,5 +31,83 @@ func Validate(stmt *ast.SelectStatement) error {
 			return fmt.Errorf("Not allowed to call aggregate functions in GROUP BY clause.")
 		}
 	}
+
+	if err := validateSRFNestedForbidden("select", stmt.Fields); err != nil {
+		return err
+	}
+	if err := validateMultiSRFForbidden("select", stmt.Fields); err != nil {
+		return err
+	}
+	return validateSRFForbidden("where", stmt.Condition)
```

Comment:
I assume SRF is only allowed in fields. If so, why only check condition here?

Reply:
Here we validate condition shouldn't has srf.

Reply:
updated.